frontend/src/individual_view.tsx

```diff
@@ -208,9 +208,23 @@ export function IndividualView({ id }: { id: string }) {
                   className={style.editButton}
                   variant="contained"
                   color="primary"
-                  onClick={() => popup(<IndividualCertificate id={id} />)}
+                  onClick={() =>
+                    popup(<IndividualCertificate id={id} action={"issue"} />)
+                  }
                 >
-                  Beställ certifikat
+                  Skapa nytt certifikat
+                </Button>
+              )}
+              {user?.canEdit(id) && (
+                <Button
+                  className={style.editButton}
+                  variant="contained"
+                  color="primary"
+                  onClick={() =>
+                    popup(<IndividualCertificate id={id} action={"update"} />)
+                  }
+                >
+                  Uppdatera certifikat
```

Comment:
After some playing and thinking, I believe we need to do this nicer, even for a MVP.

If there is a paper certificate these should not be shown at all as far as I understand (digital certificate can neither be created nor updated).

If there is a digital certificate "Uppdatera certifikat" should be available.

If there is no certificate, "Skapa nytt certifikat" should be shown.

If I understand the current design correctly, this should be possibly by testing if integer conversion of `certificate` equals integer conversion of `digital_certificate`.

Reply:
In the design stage we contemplated this options but later decided not to do this to simplify things.